design/ShaderLanguageFAQ.md

```diff
@@ -0,0 +1,158 @@
+*   It is suitable for humans to read and write.
+*   It is strongly typed.
+*   It directly represents block-structured programming, including
+    sequence, selection (if/else), and repetition (looping).
+*   It can be faithfully and simply converted to and from SPIR-V
+    which only uses features available in WebGPU.
+*   It can be faithfully and simply converted to Metal Shading Language
+    which only uses features available in WebGPU.
+*   It can be faithfully and simply converted to HLSL
+    which only uses features available in WebGPU.
+
+The W3C WebGPU community group agreed to this direction during the
+February 2020 face-to-face meeting in Redmond.
+
+
+## How do you pronounce “WGSL”?
+
+The W3C WebGPU Community Group does not prescribe a particular
+pronunciation of “WGSL”.
+
+You can say it letter-by-letter, like “W G S L” or you can say it as a single word, like “wiggzle”, “wiggles”, or “wigsel”.
+
+
+## What does a WGSL shader look like?
+
+WGSL has not yet been specified.
+However, early prototyping work indicates the language can have a look
+and feel as represented in the following examples:
+
+A simple fragment shader:
+
+
+```
+[[location 0]] var<out> gl_FragColor : vec4<f32>;
+
+fn main() -> void {
+    gl_FragColor = vec4<f32>(0.4, 0.4, 0.8, 1.0);
+    return;
+}
+entry_point fragment = main;
+```
+
+
+A code fragment showing if/else:
+
+
+```
+  if (value > 5) {
+    wvar = 14;
+  } elseif (value > 2) {
+    wvar = 12;
+  } else {
+    wvar = 0;
+  }
+```
+
+
+A code fragment showing a switch:
+
+
+```
+switch (a) {
+  case 1: {
+    # The break is implicit
+  }
+  case 2: {
+     fallthrough;
+  }
+  default: {
+  }
+}
+```
+
+
+A code fragment showing a loop:
+
+
+```
+var i : i32 = 0;
+var b : f32 = 2.4;
+loop {
+  break if (i == 4);
+
+  b = b * 3.2;
+  continuing {
+     i = i + 1;
+  }
+}
+```
+
+This syntax is still under heavy development.
+If you have ideas for improvement, or constructive criticism,
+please file issues on the [issue tracker](https://github.com/gpuweb/gpuweb/issues/new?assignees=&labels=wgsl&template=wgsl.md&title=).
+
+## Where can I find the spec?
+
+The WGSL spec will be developed on the WebGPU Community Group area on GitHub.
+See https://github.com/gpuweb/gpuweb/tree/master/wgsl
+
+
+## What browsers support WGSL?
+
+WGSL is the shader language for WebGPU.
+Any browser implementing WebGPU will be required to accept shaders in WGSL.
+Specifically, the WebGPU conformance test suite will only use shaders written
+in WGSL.
+
+Browser support for WGSL is expected to be developed over time, along
+with the development of WGSL itself.
+
+
+## Will browsers accept other shader languages for WebGPU?
+
+Accepting a shader language other than WGSL is not part of WebGPU.
+Doing so harms interoperability, as a shader in that other language is
+not portable across browsers.
+
+
+## Where can I find WGSL tooling?
+
+The WebGPU Community Group does not deliver tooling for WGSL.
```

Comment:
It's going to generally not be clear to people what the difference between the Community Group and its members outside activities are, so we should be careful about phrasing.
When we lead with "The WebGPU Community Group does not deliver tooling for WGSL", it sounds like we're not going to make tooling (because other people see "the working group" as "all the members of the working group", even though we draw a line between these.

How about:
> Although the WebGPU Community Group itself does not deliver official tooling for WGSL, WebGPU implementers are...

Reply:
"Standardization group" is probably better than "Community Group" for this audience.